pkgs/sketch_pad/lib/model.dart

```diff
@@ -309,27 +332,22 @@ class AppServices {
   }
 }
 
-int _compareIssues(AnalysisIssue a, AnalysisIssue b) {
-  final diff = a.severity - b.severity;
-  if (diff != 0) return -diff;
-
-  return a.charStart - b.charStart;
-}
-
 extension AnalysisIssueExtension on AnalysisIssue {
-  int get severity => switch (kind) {
-        'error' => 3,
-        'warning' => 2,
-        'info' => 1,
-        _ => 0,
-      };
+  int get severity {
+    return switch (kind) {
+      'error' => 3,
+      'warning' => 2,
+      'info' => 1,
+      _ => 0,
+    };
+  }
 }
 
 enum Channel {
   stable('Stable', 'https://stable.api.dartpad.dev/'),
   beta('Beta', 'https://beta.api.dartpad.dev/'),
   // This channel is only used for local development.
-  localhost('Localhost', 'http://localhost:8082/');
+  localhost('Localhost', 'http://localhost:8080/');
```

Comment:
The local version of dart_services still defaults to serving on a 8082 port. Any particular reason you changed this?

Reply:
Hmm, I did mean to switch this over to 8080 (for no particular reason) but didn't mean to diverge from what the server default to. I'll update the server code (possibly in a separate PR).